contrib/babelfishpg_tsql/src/pltsql_coerce.c

```diff
@@ -1087,6 +1100,17 @@ init_tsql_datatype_precedence_hash_tab(PG_FUNCTION_ARGS)
 	PG_RETURN_INT32(0);
 }
 
+/*
+ * Returns true if both the datatype related hash tables
+ * are fully initialised, false otherwise.
+ */
+Datum
+datatype_hash_tables_initialised(PG_FUNCTION_ARGS)
+{
+    bool inited = inited_ht_tsql_cast_info & inited_ht_tsql_datatype_precedence_info;
```

Comment:
Given that this is a test-purpose, I'd prefer to throw an error with a specific error message, e.g., "tsql cast info hash table is not initialized".

Reply:
Done!